1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -1,92 +1,89 @@
+# پلیفیل‌ها و ترنسپایلرها
```

Comment:
There's been a new line at the first line, add it back.

Reply:
@mahdyar sorry about all these. I was using a formatter and it was changing my files. Hopefully it's fixed now.

Reply:
It happens. :)